Sources/Shadow/Shadow.swift

```diff
@@ -0,0 +1,106 @@
+import SwiftUI
+
+public enum Shadow: String, CaseIterable {
```

Comment:
Remove `String` conformance, and based on your design remove `CaseIterable` if possible.

Reply:
I need the `String` for the title of the `picker` on the `demoApp`. Should I make a variable in the `DemoApp` then?

Reply:
@mohsen-schibsted Exactly, lets encapsulate the logics related to `DemoApp` inside of it.
The same as `CaseIterable`, who needs to add all the shadows to a single view in consumer side.